### DISCUSSION TITLE
PyPGx supports targeted sequencing data

### DISCUSSION THREAD

**Discussion by sbslee** [pypgx/pypgx#70] [General]
The main purpose of this discussion is to let people know that PyPGx supports targeted sequencing data (in addition to data from WGS, WES, SNP array, and long-read sequencing) for PGx genotyping including structural variation detection. For example, I recently published a paper where I applied PyPGx to the ClinPharmSeq data ([ClinPharmSeq: A targeted sequencing panel for clinical pharmacogenetics implementation](https://doi.org/10.1371/journal.pone.0272129)).

Relevant resources:
- Overview of [NGS pipeline](https://pypgx.readthedocs.io/en/latest/readme.html#ngs-pipeline) from PyPGx
- List of 59 PGx [genes](https://pypgx.readthedocs.io/en/latest/genes.html) targeted by PyPGx

**Comment by sbslee** [comment]
@AnnaGreenOrebro, I saw your recent post asking whether Cyrius is planning to support targeted sequencing data. Have you tried PyPGx? I'm asking here because I didn't want to hijack the discussion :)

**Reply by sbslee** [reply]
Same for @JARM89 and @BTHermann.

**Reply by sbslee** [reply]
Also @CYP2D.

**Reply by sbslee** [reply]
Adding @yafengwen.

**Reply by yafengwen** [reply]
@sbslee , Thanks a lot for sharing! 
We have a small sample size of targeted sequencing data for PGx and I am comparing the results from different bioinformatics tools to the traditional genotyping method. It would be interesting to try PyPGx as well!

**Reply by sbslee** [reply]
Adding @neoscreen.

**Reply by sbslee** [reply]
Adding @aroon-color from the Aldy repository.

**Reply by sbslee** [reply]
Also @anh151 and @deb0612.

**Comment by AnnaGreenOrebro** [comment]
Thanks! I will try, kind of you!
Anna

Från: sbslee ***@***.***>
Skickat: den 24 augusti 2022 07:26
Till: sbslee/pypgx ***@***.***>
Kopia: Green Anna, Klinisk genetik ***@***.***>; Mention ***@***.***>
Ämne: Re: [sbslee/pypgx] PyPGx supports targeted sequencing data (Discussion #70)


@AnnaGreenOrebro<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAnnaGreenOrebro&data=05%7C01%7Canna.green%40regionorebrolan.se%7C7cdae2c784e54f06e52f08da8591237d%7Caece5b1982274c2782181aea120ec062%7C0%7C0%7C637969155655725467%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=I%2BcY9Mi6IqzbqfEe2G03KC0jwHeOqX%2FtGP29jokqn5w%3D&reserved=0>, I saw your recent post asking whether Cyrius is planning to support targeted sequencing data. Have you tried PyPGx? I'm asking here because I didn't want to hijack the discussion :)
